_includes/fellow-list.html

```diff
@@ -1,18 +1,34 @@
 {% assign aria_id = fellow.title | url_encode | truncate: 15, "" %}
+{% assign i = 1 %}
 {% for fellow in include.fellows %}
-<a href="{{ site.baseurl }}{{ fellow.url }}" class="list-item grid-row grid-gap" aria-labelledby="{{ aria_id }}">
-  <div class="desktop:grid-col-3 tablet:grid-col-4 usa-prose list-item-img">
-    {% if fellow.image %}
-    {% asset "{{ fellow.image }}" class="circle-card" alt="{{ fellow.image_alt_text }}" %}
-    {% endif %}
+{% if i==1 %}
+<div class="grid-row">
+{% endif %}
+  <div class="fellow-card desktop:grid-col-3 tablet:grid-col-6">
+    <a href="{{ site.baseurl }}{{ fellow.url }}" aria-labelledby="{{ aria_id }}">
+      <div class="usa-prose fellow-img">
+        {% if fellow.image %}
+        {% asset "{{ fellow.image }}" class="circle-card" alt="{{ fellow.image_alt_text }}" %}
+        {% endif %}
+      </div>
+      <div>
+        <h2 class="body-intro--bold" id="{{ aria_id }}">{{ fellow.title }}</h2>
+        <p class="fellow-card--specialty">{{fellow.skills | first}}</p>
```

Comment:
This is the part that will need to change once we actually have specialties in folks' bios. Right now, I'm just showing whatever shows up first in the `skill` frontmatter.